lib/combine_pdf.rb

```diff
@@ -10,21 +10,21 @@
 # require the RC4 Gem
 require 'rc4'
 
-load 'combine_pdf/api.rb'
-load 'combine_pdf/renderer.rb'
-load 'combine_pdf/page_methods.rb'
-load 'combine_pdf/basic_writer.rb'
-load 'combine_pdf/decrypt.rb'
-load 'combine_pdf/fonts.rb'
-load 'combine_pdf/filter.rb'
-load 'combine_pdf/parser.rb'
-load 'combine_pdf/pdf_public.rb'
-load 'combine_pdf/pdf_protected.rb'
-load 'combine_pdf/exceptions.rb'
+require 'combine_pdf/api'
+require 'combine_pdf/renderer'
+require 'combine_pdf/page_methods'
+require 'combine_pdf/basic_writer'
+require 'combine_pdf/decrypt'
+require 'combine_pdf/fonts'
+require 'combine_pdf/filter'
+require 'combine_pdf/parser'
+require 'combine_pdf/pdf_public'
+require 'combine_pdf/pdf_protected'
+require 'combine_pdf/exceptions'
 
-# load "combine_pdf/operations.rb"
+# require 'combine_pdf/operations'
 
-load 'combine_pdf/version.rb'
+require 'combine_pdf/version'
```

Comment:
The speed gain is moot if you use `bootsnap`.

As for `pysch` and such, they are a special case because they are default gems so they have to worry about the other version of the gem bundled with Ruby.

So no, I don't think most gems should use `require_relative`.

Reply:
I too doubt if `require_relative` is important here, especially when loading files located in a nested folder prefixed with the gem's name.

In general I have no preference for the distinction, especially as `require 'combine_pdf'` isn't expected to be called more than once.

Reply:
Thanks to you, this was a very good chance to ask about `require` and `require_relative`.

Sometimes I use [gem_bench](https://github.com/pboling/gem_bench) to locally compare gems for performance, and `require_relative` helps a lot